lib/src/builder.dart

```diff
@@ -57,6 +61,23 @@ class ProtocBuilder implements Builder {
   final bool grpcEnabled;
   final bool useInstalledProtoc;
   final bool precompileProtocPlugin;
+  final bool wellKnownTypesEnabled;
+
+  late final List<String> wellKnownTypes = wellKnownTypesEnabled
```

Comment:
Should we turn this into a getter rather than using "late final", which has potential to throw when initialization is not correct?

```suggestion
  List<String> get wellKnownTypes = wellKnownTypesEnabled ? ...
```

Reply:
I don't think getter is a good idea here since `wellKnownTypes` is basically a constant list. 

I'd suggest making `wellKnownTypes` a static constant:
```dart
static const List<String> wellKnownTypes = [
  'google/protobuf/any.proto',
  'google/protobuf/api.proto',
  'google/protobuf/descriptor.proto',
  'google/protobuf/duration.proto',
  'google/protobuf/empty.proto',
  'google/protobuf/field_mask.proto',
  'google/protobuf/source_context.proto',
  'google/protobuf/struct.proto',
  'google/protobuf/timestamp.proto',
  'google/protobuf/type.proto',
  'google/protobuf/wrappers.proto'
];
```

Then we just check if `wellKnownTypesEnabled` is true before using it.
```dart
if (wellKnownTypesEnabled) ...wellKnownTypes;
```